src/pages/Login-ResetPassword/login-resetpassword.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { FC, useState } from 'react';
+import Axios from 'axios';
+import Footer from '../../components/Footer/footer';
+import Navbar from '../../components/NavBar/navbar';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faUser } from '@fortawesome/free-solid-svg-icons';
+import Hackmercedlogo from '../../assets/images/hackmerced-logo.png';
+
+import './login-resetpassword.scss';
+
+const LoginResetPassword: FC = (): JSX.Element => {
+    const [form, setForm] = useState({ email: "" });
+
+    const handleInputChange = (
+        event: React.ChangeEvent<HTMLInputElement> | React.ChangeEvent<HTMLSelectElement>,
```

Comment:
```suggestion
        event: React.ChangeEvent<HTMLInputElement>,
```

Remove since you are not using a select form option